app1.js

```diff
@@ -91,13 +91,11 @@ function userGuessingGame(){
             else {
                 if(userAnswer < correctAnswer){
                     alert('Your answer is too low. Try Again! You have ' + (numberOfAttempts -i) + ' attempts left');
-                } else if (userAnswer > correctAnswer){
```

Comment:
This isn't explicitly needed since the opposite of less than has to be greater than and you already checked the '=='s' case